examples/kubernetes/cluster/main.tf

```diff
@@ -1,25 +1,23 @@
-data "vkcs_kubernetes_clustertemplate" "ct" {
-  version = "1.24"
-}
-
 resource "vkcs_kubernetes_cluster" "k8s-cluster" {
-  depends_on = [
-    vkcs_networking_router_interface.k8s,
-  ]
+  name                = "k8s-cluster"
+  cluster_template_id = data.vkcs_kubernetes_clustertemplate.k8s_124.id
+  master_flavor       = data.vkcs_compute_flavor.basic.id
+  master_count        = 1
 
   labels = {
-    cloud_monitoring = "true"
-    kube_log_level   = "2"
+    cloud_monitoring         = "true"
+    kube_log_level           = "2"
```

Comment:
lets add default values to descritiptions in code. also fix GB suffix in volume size

Reply:
fixed description in optional fields
for 
Integer value, default XX (Gb).

Reply:
"in code", not in examples

Reply:
fixed